src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
```

Comment:
You can use selector `*` to reset all margin
`* {
  margin: 0;
}`
after this manipulation, all elements will have `margin 0` by default and you can delete the 25th line.